src/patternfly/components/TextInputGroup/text-input-group-group.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-text-input-group__group{{#if text-input-group-group--modifier}} {{text-input-group-group--modifier}}{{/if}}"
```

Comment:
Could you add this to the text input group docs?

Reply:
I would, but there don't seem to be any text input group docs to add to. Feels like a followup to document all of text input group.